packages/git-safe-dependencies/README.md

```diff
@@ -2,7 +2,7 @@
 
 > WARNING: This is an alpha version. The opinions are still forming and anything can change at any time. Test coverage is incomplete. Use at your own risk.
 
-Validate your `package.json` and lockfile (NPM, Yarn classic or Yarn Berry) against an opinionated set of rules around using git dependencies that we at LavaMoat recommend.  
+Validate your `package.json` and lockfile (NPM, Yarn classic or Yarn Berry) and your GitHub Actions workflow files against an opinionated set of rules around using git dependencies that we at LavaMoat recommend.  
```

Comment:
```suggestion
Validate your `package.json` and lockfile (NPM, Yarn classic or Yarn Berry) and [harden](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions) your GitHub Actions workflow files against an opinionated set of rules around using git dependencies that we at LavaMoat recommend.  
```

Reply:
nope, we're enforcing our opinions, not these from the link - it says it's ok to pin to a tag if you trust someone, which is not good enough :)

Reply:
oops yep the link is too general, meant only `Pin actions to a full length commit SHA`